README.md

```diff
@@ -36,26 +36,40 @@ of data.
 
 
 ```py
+import pandas as pd
 import matplotlib.pyplot as plt
-from gluonts.dataset.util import to_pandas
 from gluonts.dataset.pandas import PandasDataset
-from gluonts.dataset.repository.datasets import get_dataset
+from gluonts.dataset.split import split
 from gluonts.mx import DeepAREstimator, Trainer
 
-dataset = get_dataset("airpassengers")
+# Load data from a CSV file into a PandasDataset
 
-deepar = DeepAREstimator(prediction_length=12, freq="M", trainer=Trainer(epochs=5))
-model = deepar.train(dataset.train)
+url = (
```

Comment:
Should we do it in one expression?

```py
# Load data from a CSV file into a PandasDataset
dataset = PandasDataset(
    pd.read_csv(
        "https://raw.githubusercontent.com/AileenNielsen/"
        "TimeSeriesAnalysisWithPython/master/data/AirPassengers.csv",
        index_col=0,
        parse_dates=True,
    ),
    target="#Passengers"
)
```

Reply:
Yes, looks good, but I did it in to: constructing the dataframe, wrapping it into a dataset. I think this separation makes sense since it's what one would do interactively (allows to inspect the dataframe)

Reply:
Plus, it ends up even shorter :-)